doc/going-further/helpers/crypto.md

```diff
@@ -78,3 +78,16 @@ function decrypt_file(string $file): void
     decrypt_file_with_password($file, 'my super secret password');
 }
 ```
+
+> [!NOTE]
+> When no destination path is provided, the function follows these rules:
+> 
+> For files ending with `.enc` extension:
+> - The function attempts to create the decrypted file with the same name but without the `.enc` extension
+>   - Example: `secret.txt.enc` → `secret.txt`
+> - If a file already exists at that path, it uses the `.dec` extension instead to prevent data loss
+>   - Example: `secret.txt.enc` → `secret.txt.enc.dec` (when `secret.txt` already exists)
```

Comment:
Shouldn't it be named `secret.txt.dec` here?

Reply:
Yeah we could keep the logic that remove the `.enc` before adding the `.dec` 👍